browser_history/__init__.py

```diff
@@ -77,3 +79,33 @@ def get_bookmarks():
             utils.logger.info("%s", e)
     output_object.bookmarks.sort()
     return output_object
+
+
+# keep everything lower-cased
+browser_aliases = {
+    "google-chrome": "chrome",
+    "chromehtml": "chrome",
+    "chromium-browser": "chromium",
+    "msedgehtm": "edge",
+    "operastable": "opera",
+    "firefoxurl": "firefox",
+}
+
+
+def default_browser():
+    plat = utils.get_platform()
+
+    if plat == utils.Platform.LINUX:
+        default = webbrowser.get().name.lower()
+    else:
+        utils.logger.info("Default browser feature not supported on this OS")
```

Comment:
I think this should be a warning or an error.

Reply:
Should I raise an error or just log it as a warning? If someone is using it as an API it would be convenient to log and return `None` instead of throwing errors.

Reply:
Warning makes sense :+1: